style.css

```diff
@@ -170,29 +340,88 @@ img {
     align-items: center;
     justify-content: space-between;
     width: 100%;
+    text-align: left;
   }
 
   .part-one {
     margin-right: 1.3em;
     display: flex;
     flex-direction: column;
-    width: 50%;
+    flex: 0 1 50%;
   }
 
   .big-title {
     font-size: 4em;
     line-height: 1.1em;
   }
+
+  .about-you {
+    flex-direction: row;
+  }
+
+  .left-align {
+    text-align: left;
+  }
+
+  .vege-card {
+    margin: 0em 0.3em;
+  }
+
+  .meals {
+    flex-direction: row;
+  }
+
+  .refugee-block {
+    flex: 1 0 33%;
+    padding-top: 3em;
+    margin: 0em 0.3em;
+  }
+
+  .button-width {
+    width: fit-content;
+  }
+
+  .meal-block {
+    flex-direction: column;
+    margin: 0em 0.3em;
+  }
+
+  .section-three {
+    flex-direction: row;
+    padding: 0em 5em 3em 0em;
```

Comment:
shorthand

Reply:
Is this resolved? 🛩️

Reply:
If we have top and left same values in padding I don't how to to create shorthand?

Reply:
My bad, you are right - I read it incorrectly 😄

Reply:
;)